openvpn_status/utils.py

```diff
@@ -1,14 +1,18 @@
 from __future__ import absolute_import
 
+import re
 import datetime
 import collections
 import ipaddress
 
 from six import python_2_unicode_compatible as unicode_compatible
 from humanize.filesize import naturalsize
+from netaddr import EUI, mac_unix
 
 
 DATETIME_FORMAT_OPENVPN = u'%a %b %d %H:%M:%S %Y'
+RE_VIRTUAL_ADDR_MAC = re.compile(
+    ur'^{0}:{0}:{0}:{0}:{0}:{0}$'.format(u'[a-f0-9]{2}'), re.I)
```

Comment:
Maybe use `r` as prefix is enough.

Reply:
@shonenada `r` is not necessary for this regex expression. I choose to keep `u`.